docs/jwst/outlier_detection/outlier_detection_ifu.rst

```diff
@@ -5,39 +5,24 @@ Outlier Detection for IFU Data
 
 This module serves as the interface for applying ``outlier_detection`` to IFU
 observations, like those taken with NIRSpec and MIRI.  The code implements the
-basic outlier detection algorithm used with HST data, as adapted to JWST IFU
-observations.
-
-Specifically, this routine performs the following operations (modified from 
-:ref:`Default Outlier Detection Algorithm <outlier-detection-imaging>` ):
-
- * Extract parameter settings from input model and merge them with any user-provided values
-
-   - the same set of parameters available to 
-     :ref:`Default Outlier Detection Algorithm <outlier-detection-imaging>` 
-     also applies to this code
-
- * Resample all input :py:class:`~jwst.datamodels.IFUImageModel` images into 
-   :py:class:`~jwst.datamodels.IFUCubeModel` data cubes
-
-   - Resampling uses :py:class:`~jwst.cube_build.CubeBuildStep` to create 
-     :py:class:`~jwst.datamodels.IFUCubeModel` formatted data for processing
-   - Resampled cubes are written out to disk if the ``save_intermediate_results`` 
-     parameter is set to `True`
-
- * Create a median cube from the set of resampled 
-   :py:class:`~jwst.datamodels.IFUCubeModel` cubes
-
-   - The median cube is written out to disk if the ``save_intermediate_results`` 
-     parameter is set to `True`
-
- * Blot median cube to match each original 2D input image
-
-   - Resampled/blotted images are written out to disk if 
-     the ``save_intermediate_results`` parameter is set to `True`
-
- * Perform statistical comparison between blotted image and original image to identify outliers
- * Update input data model DQ arrays with mask of detected outliers
+basic outlier detection algorithm searching for pixels that are consistent outliers
+on the data after the :ref:`calwebb_spec2 <calwebb_spec2>`. After launch it was discovered the
+bad pixels on the MIRI detectors vary with time. The pixels varied from usable to unusable, and at times,
+back to usable  on a time frame that was too short (sometimes as short as 2 days)  to fold into the bad pixel mask applied
+in the :ref:`calwebb_detector1 <calwebb_detector1>`. 
+At this time it is believed that NIRSpec IFU data also have bad pixels that vary with time, though the time variation is
+still under study.
+
+An algorithm was developed to flag pixels that are  outliers when compared to their neighbors for a set of
+input files contained in association. The neighbor pixel differences are defined by the dispersion axis. For MIRI data,
```

Comment:
"... contained in *an* association."

Reply:
Not obvious to me what "The neighbor pixel differences are defined by the dispersion axis" really means. Does it perhaps mean that the differences between neighbors are computed in the direction of the dispersion axis? And hence differences between pixels in the spatial direction are ignored?

Reply:
Opposite - neighbors are in the spatial direction. I have changed the language - see if makes sense now. 

An algorithm was developed to flag pixels that are  outliers when compared to their neighbors for a set of
input files contained in an association. The neighbor pixel differences are the neighbors in spatial direction. For MIRI data, the dispersion axis is along the y-axis,  neighbor differences are found to the left and right of every
science pixel. For NIRSpec data, the dispersion axis is along the x-axis,  neighbor differences are
found to between the pixels above and below every science pixel